quadriga/Docker/docker-compose.yml

```diff
@@ -0,0 +1,74 @@
+version: '3.8'
+
+services:
+#  mysql:
+#    build:
+#      context: ./mysql
+#      dockerfile: Dockerfile
+#    image: mysql:9.0.0
+#    restart: always
+#    networks:
+#      - quadriga-network
+#    volumes:
+#      - mysql-var-data:/var/lib/mysql
+#      - mysql-etc-data:/etc
+#      - mysql-usr-data:/usr
```

Comment:
the mysql data files should be in a subfolder of the working directory (maybe `./db/data`?)

Reply:
actually, ideally there would just be one folder, e.g. `data` with subfolders fro mysql, neo4j, and mongodb